src/string_bytes.h

```diff
@@ -40,11 +40,6 @@ class StringBytes {
                                   v8::Local<v8::Value> encoding,
                                   enum encoding _default) {
       enum encoding enc = ParseEncoding(env->isolate(), encoding, _default);
```

Comment:
We should already know the encoding in the caller when we check the validity of the string. Can you pass the parsed encoding from the caller here instead of reparsing the encoding string again?

Reply:
Fixed.